core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -196,9 +226,30 @@ var ExplorationEditorTranslationTab = function() {
     waitFor.pageToFullyLoad();
   };
 
+  this.errorMessageElement = function() {
+    var errorMessage = element(by.css('div.error-message'));
```

Comment:
This isn't really the message right? It's the element still. Again, similar comment as above, why do we need the whole function? Can't you inline this call?

Reply:
Also, you have defined this element above also. You could use that directly I think...

Reply:
done